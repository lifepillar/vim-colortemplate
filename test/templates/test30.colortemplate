Options: creator=false timestamp=false backend=vim
Environments: gui 256
Full name:Test 30
Short name:test30
Author:y
Maintainer:w
helpfile
  @shortname_transp_bg
endhelpfile
Background:dark
Color:black rgb(0,0,0) 16 Black
Color:white #ffffff 255 White
Normal white black

; vim: ft=colortemplate
