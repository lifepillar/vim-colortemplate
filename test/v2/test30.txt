Colortemplate Options: creator=0 quiet=1 source_comment=0 timestamp=0 warnings=0
Full name:Test 30
Short name:test30
Author:y
Maintainer:w
documentation
  @shortname_transp_bg
enddocumentation
Background:dark
Color:black rgb(0,0,0) 16 Black
Color:white #ffffff 255 White
Normal white black

; vim: ft=colortemplate
