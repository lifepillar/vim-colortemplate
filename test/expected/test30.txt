test30_transp_bg
